### ISSUE TITLE
HDR over saturated (Host w/ HDR, Client w/o HDR)

### ISSUE THREAD

**Issue by wang1zhen** [moonlight-stream/moonlight-qt#1444]
**Describe the bug**
Over saturated (washed out) stream on client. The host is Windows 11 with OS HDR on running sunshine, and Client is also Windows 11 without a HDR monitor.

**Steps to reproduce**
1. Turn on HDR on Host
2. Enable Sunshine on Host
3. Connect to Host from Client

**Screenshots**
![image](https://github.com/user-attachments/assets/6a3dae5e-7d23-4ba4-96af-c3d158170752)

**Moonlight settings (please complete the following information)**
Enabling/disabling HDR support in Moonlight Client doesn't help

**Client PC details (please complete the following information)**
 - OS: Windows 11
 - Moonlight Version: 6.1.0
 - GPU: 780M

**Server PC details (please complete the following information)**
 - OS: Windows 11
 - Sunshine 0.23.1
 - GPU: Intel Iris Xe and Nvidia 3050 Ti

**Moonlight Logs (please attach)**

[Moonlight-1728976834.log](https://github.com/user-attachments/files/17374573/Moonlight-1728976834.log)

**Comment by wang1zhen** [comment]
BTW, turning OS HDR on/off on the Host does not help either.

**Comment by cgutman** [comment]
Yeah, this is currently expected behavior when streaming HDR content to an SDR client. Moonlight doesn't have HDR->SDR tonemapping support. The intent in the near future is that Sunshine will reconfigure the host display to match the HDR state of the client, which will result in an optimal experience where tonemapping is not required.

The libplacebo-based renderer used for Vulkan video and software decoding does support tonemapping, so you could try forcing software decoding. However, since your host GPU doesn't support AV1 encoding and the x265 decoder is slower than libdav1d, the software decoding performance may be too slow to be usable.

**Comment by wang1zhen** [comment]
Thank you for explaining!
Please pardon me for one more maybe off-topic question, does HDR->HDR streaming require hardware specification matching or re-calibration?

**Comment by cgutman** [comment]
For best results, you would probably want to calibrate for each different display that you stream to. However, Windows does have built-in calibration so it's possible that you could use that on the client-side instead of having to recalibrate the host.
